### PR TITLE
[Do Not Merge] Add skipActionRuleSet to appgw waf definition

### PR DESCRIPTION
**Pull request by jshaker**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by number213** [comment]
Please rebase your branch, there are a lot of unrelated commits here

**Comment by lirenhe** [comment]
@jshaker  could you take a look at the error for spell check?

**Comment by jshaker** [comment]
@number213 addressed commits
@lirenhe addressed spell check

**Review by anton-evseev — APPROVED** [review]
Can you also add examples for those properties?

**Review by lirenhe — APPROVED** [review]

**Review by KrisBash — APPROVED** [review]
LGTM

File: specification/network/resource-manager/Microsoft.Network/stable/2019-09-01/examples/WafPolicyGet.json [hunk 1]
```diff
@@ -83,6 +83,63 @@
...
+                  ],
+                  "operator": "Contains",
+                  "negationConditon": false,
+                  "matchValues": [
+                    "Windows"
+                  ]
+                }
+              ],
+              "action": "Skip",
+              "skippedManagedRulesets": [
```
Comment (number213) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-09-01/examples/WafPolicyGet.json]:
```suggestion
              "skippedManagedRuleSets": [
```

**Review by anton-evseev — CHANGES_REQUESTED** [review]

**Comment by lirenhe** [comment]
@jshaker, please fix the problem for model validation.

**Comment by jshaker** [comment]
These changes are being pushed to a later release cycle

**Pull request closed without merging**